src/components/BulkEnrollmentPage/BulkEnrollmentWarningModal.jsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+import {
+  ActionRow, AlertModal, Button, Icon,
+} from '@edx/paragon';
+import { Error } from '@edx/paragon/icons';
+
+import BulkEnrollButton from './BulkEnrollButton';
+
+const BulkEnrollWarningModal = ({
+  learners = [], isDialogOpen = false, onClose, onEnroll,
+}) => (
+  <AlertModal
+    title={(
+      <>
+        <Icon className={classNames('enroll-header', 'mr-1')} src={Error} />Revoked Learners Selected
+      </>
+      )}
+    isOpen={isDialogOpen}
+    footerNode={(
+      <ActionRow>
+        <Button variant="link" onClick={onClose}>Close</Button>
+        <BulkEnrollButton
+          learners={learners}
+          handleEnrollment={onEnroll}
+        />
+      </ActionRow>
+    )}
+  >
+    <>
+      {`Any learners with revoked licenses are not included. Click "Enroll" to enroll active
+          and pending learners only`}
+    </>
```

Comment:
nit: Why does this need to be in a Fragment versus using the text as the direct children?

```jsx
<AlertModal
  // ...
>
  Any learners with revoked licenses are not included. Click "Enroll" to enroll active and pending learners only
</AlertModal>
```

Reply:
agreed! think I had weird characters there before somehow, and has this fragment going on, removed!